src/test/java/uk/gov/justice/probation/courtcasematcher/restclient/model/courtcaseservice/CCSOffenceTest.java

```diff
@@ -30,6 +32,7 @@ private Offence buildOffence() {
                 .act("act")
                 .sequenceNumber(1)
                 .listNo(30)
+                .judicialResults(Collections.emptyList())
```

Comment:
I think it would be better to include a single judicial result otherwise it's not really testing that it streams the contents, it could just be defaulting to an empty list.